models/BudgetResults.v1.json

```diff
@@ -8,6 +8,12 @@
     },
     "funding_plan": {
       "$ref": "./FundingResults.v1.json"
+    },
+    "informations": {
```

Comment:
I don't think there is an "s" at the end of information.

Reply:
I have the feeling that "information" is weird here, here are a few proposition that I have in mind:
- logs
- traces
- notifications
- messages

Reply:
logs is good yes